samples/crypto/psa_tls/README.rst

```diff
@@ -143,7 +143,7 @@ The table mirrors the test setup used for TLS verification.
            - Also supports :ref:`PSA Crypto nrf_oberon driver<nrf_security_drivers_oberon>`
          * - ``nrf54h20dk/nrf54h20/cpuapp``
              ``nrf54h20dk/nrf54h20/cpurad``
-           - `SSF Crypto <CONFIG_SSF_PSA_CRYPTO_SERVICE_ENABLED_>`_
+           - `SSF Crypto <CONFIG_SSF_PSA_CRYPTO_SERVICE_ENABLED>`_
```

Comment:
```suggestion
           - SSF Crypto (set with :kconfig:option:`CONFIG_SSF_PSA_CRYPTO_SERVICE_ENABLED`)
```

Reply:
As above, put it into the comments column.

Reply:
Unless I have misunderstood the patch, this is a pre-existing issue unrelated to my PR.

Reply:
> As above, put it into the comments column.


Assuming I haven't misunderstood you, this is a pre-existing issue unrelated to my PR.

Reply:
Clarified as per https://github.com/nrfconnect/sdk-nrf/pull/23339#discussion_r2213238695